lib/services/android-debug-service.ts

```diff
@@ -174,7 +174,7 @@ class AndroidDebugService extends DebugServiceBase implements IPlatformDebugServ
 			debugerStarted = forwardsResult.indexOf(waitText) === -1;
 
 			if (!debugerStarted) {
-				sleep(500);
+				await sleep(500);
```

Comment:
Great! :)

Small typo: `debugerStarted` => `debuggerStarted`

What do you think of introducing a function wait.Until(function, condition, sleepIntervalMs) which implements a loop which sleeps for several iterations or until a condition is met? This way, we can introduce unit tests for it, reuse it and don't worry about such bugs anymore? I've seen this implemented and it was working good.